File: Terminal/Calculator.py
def add (x,y):
    return x+y

def soustraction(x,y):
    return x-y

def multiply(x,y):
    return x*y

def divide(x,y):
    return x/y

x = True
while x == True :
    print("--- MENU ---")
    print("Select operation :")
    print("1.Add : "  )
    print("2.Substract : ")
    print("3.Multiply : ")
    print("4.Divide : ")
    print("Q|q.Quit : ")

    choice = input("Enter choice(1/2/3/4/Q|q):")
    if choice == "1":
        num1 = float(input("Enter first number: "))
        num2 = float(input("Enter second number: "))
        print(num1,"+",num2,"=", add(num1,num2))
    elif choice == "2":
        num1 = float(input("Enter first number: "))
        num2 = float(input("Enter second number: "))
        print(num1,"-",num2,"=", soustraction(num1,num2))
    elif choice == "3":
        num1 = float(input("Enter first number: "))
        num2 = float(input("Enter second number: "))
        print(num1,"*",num2,"=", multiply(num1,num2))
    elif choice == "4":
        num1 = float(input("Enter first number: "))
        num2 = float(input("Enter second number: "))
        print(num1,"/",num2,"=", divide(num1,num2))
    elif choice == "Q" or "q":
        print("Quitting")
        x = False

File: Terminal/calculator.js
const prompt = require("prompt-sync")();

function add(a, b) {
  return a + b;
}
function Subtract(a, b) {
  return a - b;
}
function Multiply(a, b) {
  return a * b;
}
function Divide(a, b) {
  return a / b;
}

var x = true;
while (x) {
  console.log("--------------\n     MENU\n--------------");
  console.log("1. Add");
  console.log("2. Subtract");
  console.log("3. Multiply");
  console.log("4. Divide");
  console.log("5. Quit");
  var choice = prompt("Enter your choice: ");
  if (choice == `5`) {
    console.log(`Quitting`);

    break;
  }
  if (choice == `1` || choice == `2` || choice == `3` || choice == `4`) {
    console.log("Enter two numbers : ");
    const num1 = prompt("Enter the first number :  ");
    const num2 = prompt("Enter the second numbers : ");

    if (choice == `1`) {
      console.log(
        `--------------\nthe sum is ${add(parseInt(num1), parseInt(num2))}`
      );
    } else if (choice == `2`) {
      console.log(
        `--------------\nthe Subtraction is ${Subtract(
          parseInt(num1),
          parseInt(num2)
        )}`
      );
    } else if (choice == `3`) {
      console.log(
        `--------------\nthe Multiply is ${Multiply(
          parseInt(num1),
          parseInt(num2)
        )}`
      );
    } else if (choice == `4`) {
      if (num2 == 0) {
        console.log("ERROR DIVIDING BY 0");
      } else {
        console.log(
          `--------------\nthe division is ${Divide(
            parseInt(num1),
            parseInt(num2)
          )}`
        );
      }
    } else {
      console.log("Invalid choice. Please enter a valid choice.");
    }
  }
}
